lib/action_policy/graphql.rb

```diff
@@ -27,6 +27,11 @@ class << self
       # Defaults to `true`.
       attr_accessor :authorize_raise_exception
 
+      # Whether to raise an exeption if mutation field is not authorized
+      # or return `nil`.
+      # Defaults to `true`.
+      attr_accessor :authorize_mutation_raise_exception
```

Comment:
Actually, we don't need this option. Using `authorize: *` with mutations doesn't make sense, since it's called after the mutation executes.

I added this option initially but removed in the final implementation (but forgot to remove the method call from the field extension 😞).

Reply:
Yeah, that makes a lot of sense. Just wondering if could be any use case for post-factum authorization in mutation. Would it make sense to raise error when `authorize` is used in mutation?

Reply:
> Would it make sense to raise error when authorize is used in mutation?

IMO, yes, that would be great.

We can introduce a deprecation warning first and raise ArgumentError when `authorize: *` is used on a mutation field in the future releases.